Sources/IBLinterKit/Rules/ReuseIdentifierRule.swift

```diff
@@ -0,0 +1,55 @@
+import IBDecodable
+
+extension Rules {
+    
+    struct ReuseIdentifierRule: Rule {
+        
+        static let identifier = "reuse_identifier"
+        static let description = """
+Check that ReuseIdentifier same as class name.\\
```

Comment:
trailing two space to line break for markdown is not good i think because Xcode have `Automatically trim trailing whitespace` option that trim space for line break.
So i choose back-tick style line break for markdown.